tests/integration/sqcmds/cumulus-samples/route.yml

```diff
@@ -4645,6 +4645,30 @@ tests:
     "default", "prefix": "2001:db8:0:2::/64", "nexthopIps": ["fe80::5054:ff:fed6:9480",
     "fe80::5054:ff:fedb:b08e"], "oifs": ["swp2", "swp1"], "protocol": "bgp", "source":
     "", "preference": 20, "ipvers": 6, "action": "forward", "timestamp": 1616638470426}]'
+- command: route show --prefix=">24" --format=json
```

Comment:
In this and all subsequent tests you have used --prefix. You need to use --prefixlen.